### PR TITLE
Added resolutionator.app

### PR DESCRIPTION
**Pull request by bcomnes**
Nice, (and sofar gratis) app from many tricks for getting a resolution menubar item: http://manytricks.com/blog/?cat=34

### CODE REVIEW

File: Casks/resolutionator.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'resolutionator' do
+  version ':latest'
```
Comment (vitorgalvao) [inline Casks/resolutionator.rb]: Just `version :latest`, no quotes.

**Comment by bcomnes** [comment]
Fixed!  Woops.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by bcomnes** [comment]
> Could you please squash your commits?
Done

**Pull request merged**